dataflow/src/ops/trigger.rs

```diff
@@ -0,0 +1,223 @@
+use std::collections::HashMap;
+
+use prelude::*;
+
+use hyper::Client;
+use tokio_core::reactor::Core;
+use serde::Serialize;
+use serde_json;
+use std::thread;
+
+/// Applies the identity operation to the view. Since the identity does nothing,
+/// it is the simplest possible operation. Primary intended as a reference
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Trigger {
+    us: Option<IndexPair>,
+    src: IndexPair,
+    trigger: TriggerType,
+    key: Vec<usize>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum TriggerType {
+    // url to the blender, group name
+    GroupCreation{ url: String, group: String },
+}
+
+impl Trigger {
+    /// Construct a new Trigger operator.
+    pub fn new(src: NodeIndex, trigger: TriggerType, key: Vec<usize>) -> Trigger {
```

Comment:
What is the `key` parameter for? I guess this might become clearer when the docs for the `Trigger` operator are expanded a little.